packages/cli/src/config.rs

```diff
@@ -20,8 +20,19 @@ fn default_plugin() -> toml::Value {
 }
 
 impl DioxusConfig {
-    pub fn load() -> crate::error::Result<Option<DioxusConfig>> {
-        let Ok(crate_dir) = crate::cargo::crate_root() else { return Ok(None); };
+    pub fn load(bin: Option<PathBuf>) -> crate::error::Result<Option<DioxusConfig>> {
```

Comment:
I'm somewhat worried this will be confused with how `cargo run --bin` works. With cargo run --bin, it looks at the workspace and finds the binary with the specified name instead of accepting a path

Reply:
We'd have to intentionally check if the bin exists in the workspace members otherwise accepting a String would have the same result: ``path.join(bin)``

Would you want to:
- Fix the behavior now (I would probably need to pull in cargo_toml crate)
- Wait until later (I have wip stuff that will probably reiterate on this but it'll be some time)
- Push it out with current behavior and wait for next PR that reiterates on it
- Other?

Reply:
Great! API looks good now. One last thing, could you add something about the command to the docs (here)[https://github.com/DioxusLabs/dioxus/blob/master/packages/cli/docs/src/cmd/serve.md]

Reply:
Done 👍